crates/legacy/src/lib.rs

```diff
@@ -78,3 +80,18 @@ impl<Types: NodeType> std::fmt::Display for BuilderStateId<Types> {
         write!(f, "BuilderState({}@{})", self.parent_commitment, *self.view)
     }
 }
+
+/// References to the parent block that is extended to spawn the new builder state.
+#[derive(Debug, Clone)]
+pub struct ParentBlockReferences<TYPES: NodeType> {
```

Comment:
Looks like the same struct in `utils.rs` can be used by both legacy and marketplace code so we can remove it here.

Reply:
They exist in separate `crate`s.  So do we want the legacy builder to depend on the `marketplace`?

Reply:
Oh, I didn't realize `utils.rs` was in the marketplace crate. No need to change then.

Reply:
We could eventually create a shared crate between them.  But I think we can probably do that at a later point.

Reply:
Sounds good. Made an issue: https://github.com/EspressoSystems/marketplace-builder-core/issues/139.